dotnet/README.md

```diff
@@ -121,31 +121,29 @@ more features, such as connectors to services and Skills to perform specific act
 Unless you need to optimize which packages to include in your app, you will usually
 start by installing this meta-package first:
 
-* **Microsoft.SemanticKernel**
+* **[Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/)**
```

Comment:
This link remains bolded, whereas all the other links below have had bolding removed. Is that intentional? I think the bolding looks better in the rendered view, but either way they should be consistent

Reply:
This is the main package, that is why I kept it in bold!